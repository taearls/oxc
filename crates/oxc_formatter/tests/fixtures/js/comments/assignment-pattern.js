({
  // comment1 inside the left side of the assignment pattern
  random = Math.random,
  // comment2 inside the left side of the assignment pattern
  sqrt = Math.sqrt,
} = {}) => {};

([
  // comment1 inside the left side of the assignment pattern
  random = Math.random,
  // comment2 inside the left side of the assignment pattern
  sqrt = Math.sqrt,
] = []) => {};
