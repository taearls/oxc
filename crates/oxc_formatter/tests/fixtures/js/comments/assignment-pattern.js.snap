---
source: crates/oxc_formatter/tests/fixtures/mod.rs
---
==================== Input ====================
({
  // comment1 inside the left side of the assignment pattern
  random = Math.random,
  // comment2 inside the left side of the assignment pattern
  sqrt = Math.sqrt,
} = {}) => {};

([
  // comment1 inside the left side of the assignment pattern
  random = Math.random,
  // comment2 inside the left side of the assignment pattern
  sqrt = Math.sqrt,
] = []) => {};

==================== Output ====================
({
  // comment1 inside the left side of the assignment pattern
  random = Math.random,
  // comment2 inside the left side of the assignment pattern
  sqrt = Math.sqrt,
} = {}) => {};

([
  // comment1 inside the left side of the assignment pattern
  random = Math.random,
  // comment2 inside the left side of the assignment pattern
  sqrt = Math.sqrt,
] = []) => {};

===================== End =====================
